doc/source/serve/production-guide/kubernetes.md

```diff
@@ -30,7 +30,7 @@ Follow the [KubeRay quickstart guide](kuberay-quickstart) to:
 * Deploy a KubeRay operator
 
 ## Setting up a RayService custom resource (CR)
-Once the KubeRay controller is running, manage your Ray Serve application by creating and updating a `RayService` CR ([example](https://github.com/ray-project/kuberay/blob/release-0.5/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml)).
+Once the KubeRay controller is running, manage your Ray Serve application by creating and updating a `RayService` CR ([example](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-service.text-ml.yaml)).
```

Comment:
Can you pin these links to a specific version, so they still work even if the KubeRay repo structure changes?